src/constants/DishesData.ts

```diff
@@ -0,0 +1,46 @@
+import { CardProps } from "../models/types";
+
+import padKiMao from "../assets/images/padKiMao.png";
+import taMaLaKo from "../assets/images/taMaLaKo.png";
+import redFarm from "../assets/images/redFarm.png";
+import smokedPizza from "../assets/images/smokedPizza.png";
+
+import spicyFoodIcon from "../assets/images/spicyFoodIcon.svg";
+import veganFoodIcon from "../assets/images/veganFoodIcon.svg";
+import vegiFoodIcon from "../assets/images/vegiFoodIcon.svg";
+
+const DishesData: { title: string; cards: CardProps[] } = {
```

Comment:
the object of
`{ title: string; cards: CardProps[] }
`
should be an interface of the carousel
pay attention that the BE should be returning every section separately , so if necessary create specific interfaces